plugin.xml

```diff
@@ -163,6 +163,7 @@
     <config-file target="AndroidManifest.xml" parent="/manifest/application">
       <activity android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale"
           android:label="@string/app_name"
+          android:launchMode="singleTop"
```

Comment:
@coderReview Why are you adding ```singleTop``` here? I don't see any reason to launch this in a separate task.